### PR TITLE
ADD: eslint rules and prettier config

### PR DESCRIPTION
**Pull request by Goku-kun**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,19 +1,22 @@
 {
   "name": "DiscordOverflow",
   "version": "1.0.0",
-  "main": "index.js",
+  "main": "src/index.js",
```
Comment (shashwatrathod) [inline package.json]: I guess it should be src/main.js. Also, if possible you can add the ts pipeline to the start script as well

**Review by shashwatrathod — COMMENTED** [review]
Check out the comment in the code.

**Comment by Goku-kun** [comment]
After this request, I'll switch to using this repository as primary development repository instead of my fork.

**Comment by shashwatrathod** [comment]
I think we're good with this PR here.

**Pull request merged**